config.template

```diff
@@ -102,10 +105,11 @@
    {block_time, 60000},
    {late_block_timeout_seconds, 3600},
    {update_dir, "/opt/miner/update"},
+   {api_base_url, "https://api.helium.io/v1"},
```

Comment:
Should we change to stakejoy endpoint?

Reply:
Yeah good shout. Let's do that.

Reply:
On a second thought -- let's not do that as it's a 'Secret URL'.